apps/freelance/src/pages/Freelancer/CreateProfile2/index.tsx

```diff
@@ -0,0 +1,214 @@
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+	StyledPage,
+	Form,
+	InputContainer,
+	Wrapper,
+	TextAreaContainer,
+	ButtonsContainer,
+	InputHeader,
+	InputWrapper,
+} from "./style";
+import {
+	ThemeColors,
+	ThemeBackground,
+	Dashboard,
+	StyledTitle,
+	StyledButton,
+} from "@freelance/components";
+import { ThemeProvider } from "styled-components";
+
+interface educationProps {
+	institute: string;
+	occupation: string;
+	period: string;
+}
+
+interface workExperienceProps {
+	company: string;
+	workPosition: string;
+	period: string;
+}
+
+export function CreateProfile2() {
+	const { t } = useTranslation();
+	const [educationList, setEducationList] = useState<educationProps[]>([]);
+	const [workExperienceList, setWorkExperienceList] = useState<workExperienceProps[]>([]);
+
+	const education = {
+		institute: "",
+		occupation: "",
+		period: "",
+	};
+
+	const workExperience = {
+		company: "",
+		workPosition: "",
+		period: "",
+	};
```

Comment:
no no no, never constants in component. initialisation of this constant will be with each rerender, its very very dab practice, move above the component

Reply:
Fixed.